portability-core/src/main/java/org/dataportabilityproject/job/JobDao.java

```diff
@@ -70,11 +66,7 @@ public JobDao(PersistentKeyValueStore persistentKeyValueStore) {
 
   /** Returns the next job in unassigned state. */
   public String findNextJobPendingWorkerAssignment() {
-    String key = storage.getFirst(JobState.PENDING_WORKER_ASSIGNMENT.name());
-    if (key != null) {
-      return getIdFromKey(key);
-    }
-    return null;
+    return storage.getFirst(JobState.PENDING_WORKER_ASSIGNMENT);
```

Comment:
Now that it's one line, can we just inline where it is called and remove it from this object? How would that look?

Or do we want a WorkerJobDao which holds this and API specific methods in another one?